radio/src/gui/gui_common_arm.cpp

```diff
@@ -597,7 +597,7 @@ bool isTrainerModeAvailable(int mode)
 #elif defined(PCBXLITE)
 bool isTrainerModeAvailable(int mode)
 {
-  if (g_eeGeneral.bluetoothMode == BLUETOOTH_TRAINER && (mode == TRAINER_MODE_MASTER_BLUETOOTH || mode == TRAINER_MODE_SLAVE_BLUETOOTH))
+  if ((g_eeGeneral.bluetoothMode == BLUETOOTH_TRAINER && (mode == TRAINER_MODE_MASTER_BLUETOOTH || mode == TRAINER_MODE_SLAVE_BLUETOOTH)) || mode == TRAINER_MODE_MASTER_CPPM_EXTERNAL_MODULE)
```

Comment:
Are you sure you meant to condition that with BT mode been set to trainer ?

Reply:
Umm it's not?
That made me see some UI changes are needed too though...